pkgs/by-name/aw/aws-gate/package.nix

```
@@ -0,0 +1,55 @@
+{ lib
+, fetchFromGitHub
+, installShellFiles
+, python3
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "aws-gate";
+  version = "0.11.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "xen0l";
+    repo = pname;
+    rev = version;
+    hash = "sha256-9w2jP4s1HXf1gYiXX05Dt2iXt0bR0U48yc8h9T5M+EQ=";
+  };
+
+  patches = [
+    ./disable-bootstrap.patch
+  ];
+
```

As mentioned in the above comment, the `postPatch` hook could look like this:

```suggestion

  postPatch = ''
    rm aws_gate/bootstrap.py tests/unit/test_bootstrap.py
  '';

```

---

Implemented exactly like that